map/create-and-save-map/src/main/java/com/esri/samples/create_and_save_map/CreateAndSaveMapController.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2017 Esri.
```

Comment:
This is fine, but in the future, we shouldn't change these.

Reply:
Good to know. I thought the change to the default challenge handler would warrant seeing this as a revamped sample. I'll leave the year be next time.